mbf_costmap_nav/src/mbf_costmap_nav/costmap_navigation_server.cpp

```diff
@@ -390,8 +390,7 @@ bool CostmapNavigationServer::callServiceCheckPointCost(mbf_msgs::CheckPoint::Re
   std::string costmap_frame = costmap->getGlobalFrameID();
 
   geometry_msgs::PointStamped point;
-  if (! mbf_utility::transformPoint(*tf_listener_ptr_, costmap_frame, request.point.header.stamp,
-                                     ros::Duration(0.5), request.point, global_frame_, point))
+  if (! mbf_utility::transformPoint(*tf_listener_ptr_, costmap_frame, ros::Duration(0.5), request.point, point))
```

Comment:
minor remark here and in the changes below: please fix also the formatting (white space after '!')

Reply:
which remark?   
format fixed

Reply:
the formatting was the remark ;)